internal/scorecard/plugins/olm_tests.go

```diff
@@ -165,16 +190,49 @@ func NewOLMTestSuite(conf OLMTestConfig) *schelpers.TestSuite {
 		"Test suite checks if an operator's CSV follows best practices",
 	)
 
+	ts.AddTest(NewBundleValidationTest(conf), 1)
 	ts.AddTest(NewCRDsHaveValidationTest(conf), 1.25)
 	ts.AddTest(NewCRDsHaveResourcesTest(conf), 1)
-	ts.AddTest(NewAnnotationsContainExamplesTest(conf), 1)
+	ts.AddTest(NewCSVValidationTest(conf), 1)
 	ts.AddTest(NewSpecDescriptorsTest(conf), 1)
 	ts.AddTest(NewStatusDescriptorsTest(conf), 1)
 
 	return ts
 }
 
-// Test Implentations
+// Test Implementations
+
+// Run - implements Test interface
+func (t *BundleValidationTest) Run(ctx context.Context) *schelpers.TestResult {
+	res := &schelpers.TestResult{Test: t, MaximumPoints: 1}
+
+	if t.OLMTestConfig.Bundle == "" {
+		res.Errors = append(res.Errors, errors.New("add a 'bundle' directory which is required for this test"))
+		return res
+	}
+
+	validationLogOutput := new(bytes.Buffer)
+	logrus.SetOutput(validationLogOutput)
+	_, _, validationResults := manifests.GetManifestsDir(t.OLMTestConfig.Bundle)
+	for _, result := range validationResults {
+		if result.HasError() {
+			for _, e := range result.Errors {
+				res.Errors = append(res.Errors, fmt.Errorf("%s", e.Error()))
+			}
+			res.Log = validationLogOutput.String()
```

Comment:
Any reason we can't always set `res.Log` once at the end outside of any conditionals or loops?

Reply:
I have made the change moving it out of the loop.